packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics.ts

```diff
@@ -68,55 +69,70 @@ export default class CallDiagnosticMetrics extends StatelessWebexPlugin {
     this.callDiagnosticEventsBatcher = new CallDiagnosticEventsBatcher({}, {parent: this.webex});
   }
 
+  /**
+   * Returns the login type of the current user
+   * @returns one of 'login-ci','unverified-guest'
```

Comment:
looks like it can return with `null` as well

Reply:
fixed